docs/dev/client.rst

```diff
@@ -0,0 +1,79 @@
+############
+Client guide
+############
+
+This page describes the use of the ``NubladoClient`` class and especially its provided testing classes.
```

Comment:
These docs should mention how to install the client from PyPI and also mention the full Python namespace early rather than just ``NubladoClient``. If we can make API docs for the NubladoClient and its exceptions and models, even better.

Reply:
I do not know how to make API docs, or at least not how to autogenerate them.

Reply:
Docs restructured and improved.  No API autogen though.